src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -1171,4 +1187,36 @@ public VisitSummary getBookingVisitsSummary(final Long bookingId) {
         final var count = visitInformationRepository.countByBookingId(bookingId);
         return new VisitSummary(visit.map(VisitDetails::getStartTime).orElse(null), count > 0);
     }
+
+    @Transactional
+    @VerifyBookingAccess
+    public PersonalCareNeed addPersonalCareNeed(final Long bookingId, final PersonalCareNeed personalCareNeed) {
+
+        final var nextId = offenderRepository.getNextOffenderHealthProblemId();
```

Comment:
see comment further up about annotating the entity instead

Reply:
done